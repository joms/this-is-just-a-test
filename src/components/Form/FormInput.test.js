import React from 'react';
import { shallow } from 'enzyme';
import { FormInput } from '.';

describe('<Input>', () => {
    it('matches snapshot', () => {
        const onChange = jest.fn();
        const wrapper = shallow(
            <FormInput label="Hello, input" onChange={onChange} hint="this is a hint" error="This field is required" />
        );
        expect(wrapper).toMatchSnapshot();
    });
});
