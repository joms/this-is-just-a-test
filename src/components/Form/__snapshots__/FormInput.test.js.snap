// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input> matches snapshot 1`] = `
<Field
  horizontal={false}
>
  <Input
    error="This field is required"
    hint="this is a hint"
    label="Hello, input"
    onChange={[MockFunction]}
    placeholder=""
    value=""
  />
</Field>
`;
